src/transformStateWithClones.js

```diff
@@ -8,6 +8,36 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const transformedState = [];
+  let currentState = { ...state };
```

Comment:
Let's change the variable declaration to `const`.
Use more descriptive name. 
`stateCopy` for example